pkg/virtualKubelet/forge/services_test.go

```diff
@@ -121,6 +121,10 @@ var _ = Describe("Services Forging", func() {
 			input:               getService(corev1.ServiceTypeNodePort, ""),
 			expectedClusterIP:   BeNil(),
 			expectedServiceType: Equal(corev1.ServiceTypeNodePort),
+		}), Entry("NodePort Service (forcing remote NodePort)", remoteServiceTestcase{
+			input:               getService(corev1.ServiceTypeNodePort, ""),
+			expectedClusterIP:   BeNil(),
+			expectedServiceType: Equal(corev1.ServiceTypeNodePort),
```

Comment:
It seems to me this is a leftover, since it does not test anything in addition to the entry above.